File: week2/csharp/main/ProductShopMain.cs
using csharp.model;
using csharp.repository;

ProductRepository products = new ProductRepository();

products.Add(new Product()
{
    BarCode = "6420387582007",
    Description = "Notebook",
    Price = 1.25,
    Count = 10
});
products.Add(new Product()
{
    BarCode = "5942328200135",
    Description = "Vitamin Aqua",
    Price = 1.00,
    Count = 10
});

foreach (Product product in products.GetAll())
{
    Console.WriteLine(product);
}

File: week2/ts/src/product-shop-main.ts
import { Product } from "./model/product.model";
import { ProductRepository } from "./repository/product-repository";

const products = new ProductRepository();

products.add(new Product("6420387582007", "Notebook", 1.25, 10));
products.add(new Product("5942328200135", "Vitamin Aqua", 1.0, 10));

for (let product of products.findAll()) {
  console.log(product.toString());
}
